### PR TITLE
docs: add retro for 15.0.0-alpha1

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Timeline written up and conclusions conveyed from the retro last wednesday

cc @electron/wg-releases

### CODE REVIEW

**Review by VerteDinde — APPROVED** [review]
Looks awesome, thanks for writing this all out! 🙇‍♀️

**Review by groundwater — APPROVED** [review]
Thanks for this! Great work.

File: wg-releases/retros/15.0.0-alpha1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### Incident Causes
+
+The first and primary issue was that we failed to initiate a release on July 20th, the first release of a given branch is always a manual process and in this case there was no clear owner of the [cut release branch](https://github.com/electron/electron/projects/31#card-65436776) task.  This was also partially due to not having a clear go/no-go metric for whether all the other dependent tasks were complete, this hesitation lead to a delay.
```
Comment (malept) [inline wg-releases/retros/15.0.0-alpha1.md:14]: Grammar nit
```suggestion
The first and primary issue was that we failed to initiate a release on July 20th, the first release of a given branch is always a manual process and in this case there was no clear owner of the [cut release branch](https://github.com/electron/electron/projects/31#card-65436776) task.  This was also partially due to not having a clear go/no-go metric for whether all the other dependent tasks were complete, this hesitation led to a delay.
```

**Review by sofianguy — APPROVED** [review]
Thanks @MarshallOfSound!

**Pull request merged**